tools/schedule-migration/migrate.sh

```diff
@@ -17,7 +17,7 @@
 set -e
 
 NAMESPACES=$(kubectl get namespace | sed '1d' | awk '{print $1}')
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+export SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
After my review, I found that this line can be removed because `SCRIPT_DIR` is not used.